packages/discovery/Discovery.yml

```diff
@@ -453,6 +500,7 @@ apis:
         name: Case Management API
         description: Support Services Case Management API
         url: https://access.redhat.com/management/api/case_management/swagger.json
+        serverUrl: https://console.redhat.com
         useLocalFile: true
```

Comment:
Can we restrict the `console.redhat.com` `serverUrl` only to the ones where the `url` key has `console.redhat.com` in it? Here, for example, the `url` has `access.redhat.com` which means this API is not part of consoledot.